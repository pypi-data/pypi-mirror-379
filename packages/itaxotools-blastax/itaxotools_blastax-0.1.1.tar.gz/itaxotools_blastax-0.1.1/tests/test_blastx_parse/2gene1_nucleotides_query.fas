>Lyciasalamandra_atifi_2154999
CGTGAGCCGCTGTCACTAATCAACCTCCGGAAGAGAAATGAGGAAACTGGAGAGGAGGAGCTGGCATCCAAACTCAACAAGCACAAGGCCAAAGCAACCAGACATATTTTCCTCGTCCGTCACTCCCAGTACAACCTAGATGGGCGTGCTGATACAGACCGGACGCTAACACTGCTAGGTCGTGAGCAGGCTGAAATGACGGGCCAAAGACTGGCCAGTTTGGGGTTTAAATATGACCAGATAGTTCACTCTTCTATG---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
>Salamandra_algira_2033643
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GACCAGATAGTTCACTCTTCTATGACTAGAGCAGCAGAGACTACGGAGATAATCGGAAAGTACCTGCCTGGGGTGAAGAAAACCAGCACGGACCTGCTGAGAGAGGGTGCCCCAATCCCACCAGACCCCCCTATTTCTCACTGGAAGCCAGACGCTGTG---TACTACGAGGACGGTGCCCGTATTGAAGCTGCATTTCGACGCTTCATTCACAGAGCAGACGCCAAGCAGGAGAAGGATAGCTATGAGGTCTTCATTTGTCATGCCAATGTCATCCGCTACATCGTTTGCAGAGCTTTGCAGTTCCCACCTGAGGGCTGGCTTCGGATGTCCCTGAACAACGGCAGCATTACTTACCTGGTGATTCACCCGAATGGACGGGTGGCACTGCGAGTACTTGGGGAGACTGGGTTCATGCCTCCTGAGAAAATCACCCGCACG
>Salamandra_infraimmaculata_2840709
CGTGAGCCGCTGTCACTAATCAACCTCCGGAAGAGAAATGAGGAAACTGGAGAGGAGGAGCTGGCATCCAAACTCAACAAGCACAAGGCCAAAGCAACCAGACATATTTTCCTCGTCCGTCACTCCCAGTACAACCTAGATGGGCGTGCTGATACAGACCGGACGCTAACACTGCTAGGTCGTGAGCAGGCTGAAATGACGGGCCAAAGACTGGCCAGTTTGGGGTTTAAATATGACCAGATAGTTCACTCTTCTATGACTAGAGCAGCAGAGACTACGGAGATAATCGGAAAGTACCTGCCTGGGGTGAAGAAAACCAGCACGGACCTGCTGAGAGAGGGTGCCCCAATCCCACCAGACCCCCCTATTTCTCACTGGAAGCCAGACGCTGTG---TACTACGAGGACGGTGCCCGTATTGAAGCTGCATTTCGACGCTTCATTCACAGAGCAGACGCCAAGCAGGAGAAGGATAGCTATGAGGTCTTCATTTGTCATGCCAATGTCATCCGCTACATCGTTTGCAGAGCTTTGCAGTTCCCACCTGAGGGCTGGCTTCGGATGTCCCTGAACAACGGCAGCATTACTTACCTGGTGATTCACCCGAATGGACGGGTGGCACTGCGAGTACTTGGGGAGACTGGGTTCATGCCTCCTGAGAAAATCACCCGCACG
>Salamandra_lanzai_2423292
CGTGAGCCGCTGTCACTAATCAACCTCCGGAAGAGAAATGAGGAAACTGGAGAGGAGGAGCTGGCATCCAAACTCAACAAGCACAAGGCCAAAGCAACCAGACATATTTTCCTCGTCCGTCACTCCCAGTACAACCTAGATGGGCGTGCTGATACAGACCGGACGCTAACACTGCTAGGTCGTGAGCAGGCTGAAATGACGGGCCAAAGACTGGCCAGTTTGGGGTTTAAATATGACCAGATAGTTCACTCTTCTATGACTAGAGCAGCAGAGACTACGGAGATAATCGGAAAGTACCTGCCTGGGGTGAAGAAAACCAGCACGGACCTGCTGAGAGAGGGTGCCCCAATCCCACCAGACCCCCCTATTTCTCACTGGAAGCCAGACGCTGTG---TACTACGAGGACGGTGCCCGTATTGAAGCTGCATTTCGACGCTTCATTCACAGAGCAGACGCCAAGCAGGAGAAGGATAGCTATGAGGTCTTCATTTGTCATGCCAATGTCATCCGCTACATCGTTTGCAGAGCTTTGCAGTTCCCACCTGAGGGCTGGCTTCGGATGTCTCTGAACAACGGCAGCATTACTTACCTGGTGATTCACCCGAATGGACGGGTGGCACTGCGAGTACTTGGGGAGACTGGGTTCATGCCTCCTGAGAAAATCACCCGCACG
>Salamandra_salamandra_Galtier_2489775
------------------------------------------------------------CTGGCATCCAAACTCAACAAGCACAAGGCCAAAGCAACCAGACATATTTTCCTCATCCGTCACTCCCAGTACAACCTAGATGGGCGTGCTGATACAGACCGGACGCTAACACTGCTAGGTCGTGAGCAGGCTGAAATGACGGGCCAAAGACTGGCCAGTTTGGGGTTTAAATATGACCAGATAGTTCACTCTTCTATGACTAGAGCAGCAGAGACTACGGAGATAATCGGAAAGTACCTGCCTGGGGTGAAGAAAACCAGCACGGACCTGCTGAGAGAGGGTGCCCCAATCCCACCAGACCCCCCTATTTCTCACTGGAAGCCAGACGCTGTG---TACTACGAGGACGGTGCCCGTATTGAAGCTGCATTTCGACGCTTCATTCACAGAGCAGACGCCAAGCAGGAGAAGGATAGCTATGAGGTCTTCATTTGTCATGCCAATGTCATCCGCTACATCGTTTGCAGAGCTTTGCAGTTCCCACCTGAGGGCTGGCTTCGGATGTCCCTGAACAACGGCAGCATTACTTACCTGGTGATTCACCCGAATGGACGGGTGGCACTGCGAGTACTTGGGGAGACTGGGTTCATGCCTCCTGAGAAAATCACCCGCACG
>Salamandra_salamandra_Miguel_2784579
CGTGAGCCGCTGTCACTAATCAACCTCCGGAAGAGAAATGAGGAAACTGGAGAGGAGGAGCTGGCATCCAAACTCAACAAGCACAAGGCCAAAGCAACCAGACATATTTTCCTCATCCGTCACTCCCAGTACAACCTAGATGGGCGTGCTGATACAGACCGGACGCTAACACTGCTAGGTCGTGAGCAGGCTGAAATGACGGGCCAAAGACTGGCCAGTTTGGGGTTTAAATATGACCAGATAGTTCACTCTTCTATGACTAGAGCAGCAGAGACTACGGAGATAATCGGAAAGTACCTGCCTGGGGTGAAGAAAACCAGCACGGACCTGCTGAGAGAGGGTGCCCCAATCCCACCAGACCCCCCTATTTCTCACTGGAAGCCAGACGCTGTG---TACTACGAGGACGGTGCCCGTATTGAAGCTGCATTTCGACGCTTCATTCACAGAGCAGACGCCAAGCAGGAGAAGGATAGCTATGAGGTCTTCATTTGTCATGCCAATGTCATCCGCTACATCGTTTGCAGAGCTTTGCAGTTCCCACCTGAGGGCTGGCTTCGGATGTCCCTGAACAACGGCAGCATTACTTACCTGGTGATTCACCCGAATGGACGGGTGGCACTGCGAGTACTTGGGGAGACTGGGTTCATGCCTCCTGAGAAAATCACCCGCACG
>Salamandra_bernardezi
CGTGAGCCGCTGTCACTAATCAACCTCCGGAAGAGAAATGAGGAAACTGGAGAGGAGGAGCTGGCATCCAAACTCAACAAGCACAAGGCCAAAGCAACCAGACATATTTTCCTCGTCCGTCACTCCCAGTACAACCTAGATGGGCGTGCTGATACAGACCGGACGCTAACACTGCTAGGTCGTGAGCAGGCTGAAATGACGGGCCAAAGACTGGCCAGTTTGGGGTTTAAATATGACCAGATAGTTCACTCTTCTATGACTAGAGCAGCAGAGACTACGGAGATAATCGGAAAGTACCTGCCTGGGGTGAAGAAAACCAGCACGGACCTGCTGAGAGAGGGTGCCCCAATCCCACCAGACCCCCCTATTTCTCACTGGAAGCCAGACGCTGTGCAGTACTACGAGGACGGTGCCCGTATTGAAGCTGCATTTCGACGCTTCATTCACAGAGCAGACGCCAAGCAGGAGAAGGATAGCTATGAGGTCTTCATTTGTCATGCCAATGTCATCCGCTACATCGTTTGCAGAGCTTTGCAGTTCCCACCTGAGGGCTGGCTTCGGATGTCCCTGAACAACGGCAGCATTACTTACCTGGTGATTCACCCGAATGGACGGGTGGCACTGCGAGTACTTGGGGAGACTGGGTTCATGCCTCCTGAGAAAATCACCCGCACG
>Salamandra_longirostris
CGTGAGCCGCTGTCACTAATCAACCTCCGGAAGAGAAATGAGGAAACTGGAGAGGAGGAGCTGGCATCCAAACTCAACAAGCACAAGGCCAAAGCAACCAGACATATTTTCCTCATCCGTCACTCCCAGTACAACCTAGATGGGCGTGCTGATACAGACCGGACGCTAACACTTCTAGGTCGTGAGCAGGCTGAAATGACGGGCCAAAGACTGGCCAGTTTGGGGTTTAAATATGACCAGATAGTTCACTCTTCTATGACTAGAGCAGCAGAGACTACGGAGATAATCGGAAAGTACCTGCCTGGGGTGAAGAAAACCAGCACGGACCTGCTGAGAGAGGGTGCCCCAATCCCACCAGACCCCCCTATTTCTCACTGGAAGCCAGACGCTGTG---TACTACGAGGACGGTGCCCGTATTGAAGCTGCATTTCGACGCTTCATTCACAGAGCAGACGCCAAGCAGGAGAAGGATAGCTATGAGGTCTTCATTTGTCATGCCAATGTCATCCGCTACATCGTTTGCAGAGCTTTGCAGTTCCCACCTGAGGGCTGGCTTCGGATGTCCCTGAACAACGGCAGCATTACTTACCTGGTGATTCACCCGAATGGACGGGTGGCACTGCGAGTACTTGGGGAGACTGGGTTCATGCCTCCTGAGAAAATCACCCGCACG
