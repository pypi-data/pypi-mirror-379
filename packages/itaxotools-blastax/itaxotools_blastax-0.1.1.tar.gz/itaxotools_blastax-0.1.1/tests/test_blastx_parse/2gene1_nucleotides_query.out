86	98.837	Lyciasalamandra_atifi_2154999	fifth_matching_sequence_from_database_revcomp_non	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSM	1	0
86	98.837	Lyciasalamandra_atifi_2154999	second_matching_sequence_from_database_readingfra	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSM	1	0
86	98.837	Lyciasalamandra_atifi_2154999	matching_sequence_from_database_path_1:053_3:5482	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSM	1	0
86	95.349	Lyciasalamandra_atifi_2154999	fourth_matching_sequence_from_database_revcomp	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSM	1	0
86	95.349	Lyciasalamandra_atifi_2154999	third_matching_sequence_from_database	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSM	1	0
146	100.000	Salamandra_algira_2033643	matching_sequence_from_database_path_1:053_3:5482	DQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAVYYEDGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
146	97.260	Salamandra_algira_2033643	fourth_matching_sequence_from_database_revcomp	DQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
146	97.260	Salamandra_algira_2033643	third_matching_sequence_from_database	DQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
53	100.000	Salamandra_algira_2033643	fifth_matching_sequence_from_database_revcomp_non	DQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
53	100.000	Salamandra_algira_2033643	second_matching_sequence_from_database_readingfra	DQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
224	99.554	Salamandra_infraimmaculata_2840709	matching_sequence_from_database_path_1:053_3:5482	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAVYYEDGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.429	Salamandra_infraimmaculata_2840709	fourth_matching_sequence_from_database_revcomp	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.429	Salamandra_infraimmaculata_2840709	third_matching_sequence_from_database	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
131	99.237	Salamandra_infraimmaculata_2840709	fifth_matching_sequence_from_database_revcomp_non	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
131	99.237	Salamandra_infraimmaculata_2840709	second_matching_sequence_from_database_readingfra	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
224	99.554	Salamandra_lanzai_2423292	matching_sequence_from_database_path_1:053_3:5482	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAVYYEDGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.429	Salamandra_lanzai_2423292	fourth_matching_sequence_from_database_revcomp	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.429	Salamandra_lanzai_2423292	third_matching_sequence_from_database	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
131	99.237	Salamandra_lanzai_2423292	fifth_matching_sequence_from_database_revcomp_non	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
131	99.237	Salamandra_lanzai_2423292	second_matching_sequence_from_database_readingfra	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
204	100.000	Salamandra_salamandra_Galtier_2489775	matching_sequence_from_database_path_1:053_3:5482	LASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAVYYEDGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
204	97.059	Salamandra_salamandra_Galtier_2489775	fourth_matching_sequence_from_database_revcomp	LASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
204	97.059	Salamandra_salamandra_Galtier_2489775	third_matching_sequence_from_database	LASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
111	100.000	Salamandra_salamandra_Galtier_2489775	fifth_matching_sequence_from_database_revcomp_non	LASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
111	100.000	Salamandra_salamandra_Galtier_2489775	second_matching_sequence_from_database_readingfra	LASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
224	100.000	Salamandra_salamandra_Miguel_2784579	matching_sequence_from_database_path_1:053_3:5482	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAVYYEDGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.875	Salamandra_salamandra_Miguel_2784579	fourth_matching_sequence_from_database_revcomp	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.875	Salamandra_salamandra_Miguel_2784579	third_matching_sequence_from_database	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
131	100.000	Salamandra_salamandra_Miguel_2784579	fifth_matching_sequence_from_database_revcomp_non	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
131	100.000	Salamandra_salamandra_Miguel_2784579	second_matching_sequence_from_database_readingfra	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
225	99.111	Salamandra_bernardezi	matching_sequence_from_database_path_1:053_3:5482	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV-YYEDGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
225	96.000	Salamandra_bernardezi	fourth_matching_sequence_from_database_revcomp	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAV-YYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
225	96.000	Salamandra_bernardezi	third_matching_sequence_from_database	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAV-YYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
131	99.237	Salamandra_bernardezi	fifth_matching_sequence_from_database_revcomp_non	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
131	99.237	Salamandra_bernardezi	second_matching_sequence_from_database_readingfra	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
224	100.000	Salamandra_longirostris	matching_sequence_from_database_path_1:053_3:5482	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAVYYEDGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.875	Salamandra_longirostris	fourth_matching_sequence_from_database_revcomp	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
224	96.875	Salamandra_longirostris	third_matching_sequence_from_database	REPMSLINLRKRNEETGEEELASKLYKHKAKATRHKFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPSSHEKPDAVYYDHGARIEAAFRRFIHRADAKQEKDSYEVFICHANVIRYIVCRALQFPPEGWLRMSLNNGSITYLVIHPNGRVALRVLGETGFMPPEKITRT	1	0
131	100.000	Salamandra_longirostris	fifth_matching_sequence_from_database_revcomp_non	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
131	100.000	Salamandra_longirostris	second_matching_sequence_from_database_readingfra	REPLSLINLRKRNEETGEEELASKLNKHKAKATRHIFLIRHSQYNLDGRADTDRTLTLLGREQAEMTGQRLASLGFKYDQIVHSSMTRAAETTEIIGKYLPGVKKTSTDLLREGAPIPPDPPISHWKPDAV	1	0
